ophyd/_pyepics_shim.py

```diff
@@ -72,9 +72,12 @@ def add_callback(self, callback=None, index=None, run_now=False,
                                     run_now=run_now,
                                     with_ctrlvars=with_ctrlvars, **kw)
 
-    def put(self, value, wait=False, timeout=30.0, use_complete=False,
+    def put(self, value, wait=False, timeout=None, use_complete=False,
```

Comment:
This change is not really significant, because higher layers always pass down an explicit value now.